Described ibexa:content:remove-duplicate-fields in the 4.6.13 update doc (#2530)
ibexa/documentation-developer: docs/update_and_migration/from_4.6/update_from_4.6.md

## v4.6.12

If the new bundle `ibexa/core-search` has not been added by the recipes, enable it by adding the following line in `config/bundles.php`:

```php
    Ibexa\Bundle\CoreSearch\IbexaCoreSearchBundle::class => ['all' => true],
```

## v4.6.13

This release comes with a command to clean up the duplicated entries in the `ezcontentobject_attribute` table, caused by the issue described in [IBX-8562](https://issues.ibexa.co/browse/IBX-8562).

> If you're affected you can remove the duplicated entries by running the `ibexa:content:remove-duplicate-fields` command:

Reviewer: ```suggestion
If you're affected you can remove the duplicated entries by running the following command:
```
I think that it's not needed to repeat it.

``` bash
php bin/console ibexa:content:remove-duplicate-fields
```

!!! caution

    Remember about [**proper database backup**](backup.md) before running the command in the production environment.

You can customize the behavior of the command with the following options:

> - `batch-size` or `b` - number of attributes affected per iteration. Default: 10000.

Reviewer: ```suggestion
- `batch-size` or `b` - number of attributes affected per iteration. Default value = 10000.
```
To make it consistent with existing docs (example: https://doc.ibexa.co/projects/userguide/en/latest/getting_started/dashboard/dashboard_block_reference/).

> - `max-iterations` or `i` - max. iterations count (default or -1: unlimited). Default: -1.

Reviewer: ```suggestion
- `max-iterations` or `i` - max. iterations count (default or -1: unlimited). Default value = -1.
```

> - `sleep` or `s` - wait time between iterations, in milliseconds. Default: 0.

Reviewer: ```suggestion
- `sleep` or `s` - wait time between iterations, in milliseconds. Default value = 0.
```
